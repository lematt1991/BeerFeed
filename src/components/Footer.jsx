import React, {Component} from 'react';
import {Row} from 'react-bootstrap';

export default class Footer extends Component{
	render(){
		return(
			<footer class="site-footer">
		      <div class="container">
		      	<Row md={8}>
			        <p class="text-muted">Powered by Untappd</p>
			    </Row>
		      </div>
		    </footer>
		);
	}
}