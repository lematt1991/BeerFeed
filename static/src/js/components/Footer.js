import React, {Component} from 'react';


export default class Footer extends Component{
	constructor(){
		super();
	}

	render(){
		return(
			<footer class="footer">
		      <div class="container">
		        <p class="text-muted">Powered by Untappd</p>
		      </div>
		    </footer>
		);
	}

}
