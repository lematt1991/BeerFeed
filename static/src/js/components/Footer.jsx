import React, {Component} from 'react';
import {Link} from 'react-router';
import {Row} from 'react-bootstrap';

export default class Footer extends Component{
	constructor(){
		super();
	}

	render(){
		return(
			<footer class="footer">
		      <div class="container">
		      	<Row md={8}>
			        <p class="text-muted">Powered by Untappd</p>
			    </Row>
		      </div>
		    </footer>
		);
	}

}

const styles={
	link : {
	}
}
