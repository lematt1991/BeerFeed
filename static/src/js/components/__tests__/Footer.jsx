import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import sinon from 'sinon'

import Footer from '../Footer'

describe('<Footer/>', () => {
	it('mounts', () => {
		var wrapper = shallow(<Footer/>)

		expect(toJson(wrapper)).toMatchSnapshot()
	})
})