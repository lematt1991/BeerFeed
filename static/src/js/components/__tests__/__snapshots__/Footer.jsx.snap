exports[`<Footer/> mounts 1`] = `
<footer
  className="site-footer">
  <div
    className="container">
    <Row
      bsClass="row"
      componentClass="div"
      md={8}>
      <p
        className="text-muted">
        Powered by Untappd
      </p>
    </Row>
  </div>
</footer>
`;
